packages/config/tsconfig.json

```diff
@@ -1,6 +1,3 @@
 {
-  "extends": "../../tsconfig.json",
-  "compilerOptions": {
-    "rootDir": "./src"
-  }
+  "extends": "../../tsconfig.json"
```

Comment:
was having more compilation issues due to ts

Reply:
🤷